### PR TITLE
Simple cheap reservations

### PR DESCRIPTION
**Pull request by maptuhec**

### CODE REVIEW

File: .env.staging [hunk 1]
```diff
@@ -6,11 +6,12 @@ REACT_APP_domainPrefix=rc
...
-REACT_APP_LOCExchange=0xc18204e5f24006d4b86b021dae83909b527c1aa2
-REACT_APP_LOCTokenContract=0x3cdea5380de712537f32a20288723cb1ce327dd3
-REACT_APP_HotelReservationFactoryProxy=0x53d27ce5843ce0e007ce3765646bf0de32216b5a
-REACT_APP_SimpleHotelReservation=0xbb832ef99c9588c320579057f8dc26a87105d3cf
+REACT_APP_LOCExchange=0x517c58fae26e50cf57ff155f887295e0ddb60dfe
+REACT_APP_LOCTokenContract=0x9903b1a046444e4893bc292e8758326b2efa71b9
+REACT_APP_HotelReservationFactoryProxy=0xbb0fee7c523705da93c6319ba38ef4b76a9a6a97
+REACT_APP_SimpleHotelReservation=0xd567c6308e7a864d9eaf3c20d5ebf22e3a29fa3c
```
Comment (thcrnk) [inline .env.staging]: Why we have 2 simple reservations contracts

File: src/services/blockchain/validators/reservationValidators.js [hunk 2]
```diff
@@ -119,6 +121,25 @@ export class ReservationValidators {
     return true;
   }
 
+  static async validateSimpleReservationParams(jsonObj, password, reservationCostLOC, withdrawDate) {
```
Comment (thcrnk) [inline src/services/blockchain/validators/reservationValidators.js]: withdrawDate is withdrawDateInDays or something like this

File: src/services/blockchain/validators/reservationValidators.js [hunk 3]
```diff
@@ -184,6 +205,15 @@ export class ReservationValidators {
     }
   }
 
+  static validateWithdrawDateInDays(withdrawDate) {
+    const nowDaysFormatted = formatTimestampToDays(new Date().getTime() / 1000 | 0);
+    let day = 60 * 60 * 24;
+    let yearsPeriod = ((day * 356) + 2) * yearsForTimeValidation;
```
Comment (thcrnk) [inline src/services/blockchain/validators/reservationValidators.js]: days in year are 365 not 356. Also move it to the constants

File: src/services/blockchain/hotelReservation.js [hunk 4]
```diff
@@ -188,13 +190,41 @@ export class HotelReservation {
...
       recipientAddress,
       overrideOptions
     );
 
+    return createSimpleHotelReservationTxResult;
+  }
+
+  static async createSimpleReservation(jsonObj, password, reservationCostLOC, withdrawDate) {
+
+    const withdrawDateFormatted = formatTimestampToDays(withdrawDate);
```
Comment (thcrnk) [inline src/services/blockchain/hotelReservation.js]: We are expecting withdrawDate to be in seconds. Add comment or change the name to withdrawDateSeconds

File: .env.staging (hunk 1 above)
Comment (maptuhec) [reply .env.staging]: They are two different contracts. This is discussed internally.

File: src/services/blockchain/validators/reservationValidators.js (hunk 3 above)
Comment (maptuhec) [reply src/services/blockchain/validators/reservationValidators.js]: This is something that will never be changed, do we need this to be moved outside of the function ? It is actually a constant, but we use the file with the constants to easily update the values.

**Pull request merged**